src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -200,12 +159,9 @@ export const ConversationPaginationContainer = createPaginationContainer(
           edges {
             node {
               id
```

Comment:
In an ideal world I'd be able to remove this edge definition altogether (because it's contained in `ConversationMessages_messages` but relay compiler didn't really like that so we had to leave _something_.